ray_beam_runner/portability/execution.py

```diff
@@ -57,6 +59,7 @@ def ray_execute_bundle(
     transform_buffer_coder: Mapping[str, typing.Tuple[bytes, str]],
     expected_outputs: translations.DataOutput,
     stage_timers: Mapping[translations.TimerFamilyId, bytes],
+    split_manager,
```

Comment:
I didn't see we update the call to ray_execute_bundle() with this new split_manager in this patch? Did i miss anything?